tasties_app/tests/conftest.py

```diff
@@ -188,3 +188,61 @@ def signed_up_credentials():
 @pytest.fixture()
 def login_to_site(client):
     client.post('/login/', data={'username': VALID_USER, 'password': VALID_PASSWORD})
+
+
+@pytest.fixture
+def create_user():
+    """
+    This fixture create a user for test the recipe_view and the add_comment methods.
+    """
+    user = User(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
```

Comment:
There's already a fixture `signed_up_credentials` in `conftest.py` that creates this exact user.